stable/redis-ha/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 2.0.1
+version: 2.0.2
```

Comment:
since this is a new feature, I recommend bumping to 2.1.0

Reply:
done